defs.mak

```diff
@@ -61,6 +61,10 @@ ifeq ($(MYST_RELEASE),1)
 OPTIMIZATION_CFLAGS += -O3
 endif
 
+ifdef MYST_ENABLE_GCOV
+GCOV_CFLAGS += -O0 --coverage -fprofile-arcs -ftest-coverage
```

Comment:
Isn't `--coverage` the same as `-fprofile-arcs -ftest-coverage`?

Reply:
I believe so, thanks for pointing it out. I was trying something out, now removed it.